app/assets/javascripts/map/views/LayersCountryView.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * The layers filter module.
+ *
+ * @return singleton instance of layers fitler class (extends Backbone.View).
+ */
+define([
+  'backbone',
+  'underscore',
+  'chosen',
+  'map/presenters/LayersCountryPresenter',
+  'map/collections/CountryCollection',
+  'handlebars',
+  'text!map/templates/layersCountry.handlebars',
+], function(Backbone, _, chosen, Presenter, CountryCollection, Handlebars, tpl) {
+
+  'use strict';
+
+  var LayersCountryModel = Backbone.Model.extend({});
+
+
+  var LayersCountryView = Backbone.View.extend({
+
+    el: '#country-layers',
+
+    template: Handlebars.compile(tpl),
+
+    model: new (Backbone.Model.extend({
+      country: null,
+      countryName: null,
+      countryLayers: null
+    })),
+
+    events: {
+      'change #layers-country-select' : 'changeIso',
+      'click #layers-country-reset' : 'resetIso'
+    },
+
+    initialize: function(map, countries) {
+      // Init presenter
+      this.presenter = new Presenter(this);        
+      this.map = map;
+      this.countries = countries;
+      
+      this.render();
+      this.listeners();
+    },
+
+    render: function() {
+      this.$el.html(this.template({
+        countries: this.countries.toJSON(),
+        country: this.model.get('country'),
+        countryName: this.model.get('countryName') || 'Country',
+        countryLayers: this.model.get('countryLayers')
+      }));
+      
+      this.cache();
+      this.chosen();
+      this.legibility();
+    },
+
+    cache: function() {
+      this.$select = this.$el.find('#layers-country-select');
+      this.$selectChosenId = '#layers_country_select_chosen';
+      // This belongs to the parent, is there a better way to do this?
```

Comment:
Maybe we can move the `legibility` function to the parent view? If it is dependent on this view having rendered, you could potentially have this view trigger an event when it's rendered so the parent can resize, or inject a method from the parent.